Generate content of the knowledge base index page (#794)
Avaiga/taipy-doc: docs/knowledge_base/demos/background_remover.md

---
title: Background Remover
category: demos
type: code
data-keywords: gui ai

> short-description: An interactive interface, and efficient Python code to easily remove image backgrounds with automatic detection.

Reviewer: ```suggestion
short-description: An interactive application to easily remove image backgrounds using automatic detection.
```

img: images/background-remover.png

> ---

Author: To reviewers: These are the meta-data. They have to be properly added to each new item to add.
Please feel free to quickly review the others. They are very similar, so that should not take so long.
